### DISCUSSION TITLE
Include --version parameter

### DISCUSSION THREAD

**Discussion by ThomasArdal** [spectreconsole/spectre.console#1202] [Ideas]
Feature request for the Spectre.Console.Cli. It would be nice to have a -v / --version option that would output the version of the current assembly to the screen. Maybe even make it configurable where to pull the version. I currently just add a version property on my main command and check for any value as the first thing. But it would be better to have this built-in somehow (IMO).

**Comment by seesharprun** [comment]
There seems to be a relevant issue related to this: https://github.com/spectreconsole/spectre.console/issues/1216

The issue shows where in the code it checks for a default command before supporting the `--version` parameter.

**Comment by FrankRay78** [comment]
Agreed.  https://github.com/spectreconsole/spectre.console/issues/1216 has now been fixed.
